aeron-driver/src/main/c/aeron_publication_image.h

```diff
@@ -313,7 +313,11 @@ inline int64_t aeron_publication_image_join_position(aeron_publication_image_t *
 
     for (size_t i = 0, length = image->conductor_fields.subscribable.length; i < length; i++)
     {
-        int64_t sub_pos = aeron_counter_get_volatile(image->conductor_fields.subscribable.array[i].value_addr);
+        aeron_tetherable_position_t *tetherable_position = &image->conductor_fields.subscribable.array[i];
+        if (tetherable_position->state == AERON_SUBSCRIPTION_TETHER_RESTING)
```

Comment:
We try to always use braces as a code style and put the constant on the left to prevent accidental assignment.

Reply:
Fixed, always wondered why the constant on the left, that makes sense.